Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -1747,37 +1748,13 @@ namespace ScriptCanvasEditor
             ScriptCanvasEditor::SourceHandle& fileAssetId = memoryAsset;
             int currentTabIndex = m_tabBar->currentIndex();
 
-            AZ::Data::AssetId oldId = fileAssetId.Id();
             AZ::Data::AssetInfo assetInfo;
             assetInfo.m_assetId = fileAssetId.Id();
             AZ_VerifyWarning("ScriptCanvas", AssetHelpers::GetAssetInfo(fileAssetId.Path().c_str(), assetInfo)
                 , "Failed to find asset info for source file just saved: %s", fileAssetId.Path().c_str());
 
-            const bool assetIdHasChanged = assetInfo.m_assetId.m_guid != fileAssetId.Id();
             fileAssetId = SourceHandle(fileAssetId, assetInfo.m_assetId.m_guid, fileAssetId.Path());
 
-            // check for saving a graph over another graph with an open tab
-            for (;;)
```

Comment:
These lines of code repeatedly issued a false negative warning about failing to find a graph it just opened.